src/action_runner/action_runner.cpp

```diff
@@ -115,7 +115,17 @@ bool install_new_version_stage_2(std::wstring installer_path, std::wstring_view
         sei.fMask = { SEE_MASK_FLAG_NO_UI | SEE_MASK_NOASYNC | SEE_MASK_NOCLOSEPROCESS | SEE_MASK_NO_CONSOLE};
         sei.lpFile = installer_path.c_str();
         sei.nShow = SW_SHOWNORMAL;
-        sei.lpParameters = L"-silent";
+        std::wstring parameters = L"--no_full_ui";
+        if(launch_powertoys)
```

Comment:
The auto formatter is not working for action_runner.cpp?

Reply:
as long as I don't forget to run it :)